app/models/transaction_log_entry.rb

```diff
@@ -92,6 +92,7 @@ def self.product_partners(product_id)
   def self.product_partners_with_balances(product_id)
     User.joins('inner join transaction_log_entries tle on tle.wallet_id = users.id').
       where(tle: {product_id: product_id}).
+      having('sum(cents) > 0').
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.